cogeo_mosaic/create.py

```diff
@@ -12,44 +11,40 @@
 
 
 def create_mosaic(
-    dataset_list: Sequence[str],
```

Comment:
Should this be changed back to `Sequence`? I think I might've missed that in the merge conflicts

Reply:
I'll fix that

Reply:
I'll just push it back

Reply:
Back to `Sequence`